packages/react-components/react-datepicker/src/components/DatePicker/useDatePickerStyles.ts

```diff
@@ -33,11 +34,6 @@ const useTextFieldStyles = makeStyles({
     '& input[readonly]': {
       cursor: 'pointer',
     },
-    input: {
-      '::-ms-clear': {
-        display: 'none',
-      },
-    },
```

Comment:
This was unused and made griffel complain since we are creating a style inside a style.